_data/changelogs/component-header.yml

```diff
@@ -2,6 +2,15 @@ title: Header
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Removed the CSS order property from mobile header navigation.
+    summaryAdditional: Now, the visual order of the component matches the tab order. If you would like to visually keep the search bar at the top of the menu, you will need to reorder your markup in the mobile view.
```

Comment:
```suggestion
    summaryAdditional: Now, the visual order of the component matches the tab order. This is marked as a breaking change, but you only need to change your project code if you want to visually keep the search bar at the top of the menu.
```

Reply:
Do we want `visually keep` here? Or should it be `keep` instead for clarity?

Reply:
Oh good point. "Visually keep" doesn't paint the full picture since it ignores that it will change the non-visual experience. It is a bit nuanced since non-visually nothing has changed (so "move" might be a more accurate verb), but I think that most users would find "keep" to be the clearest. 

Want me to go ahead and make the update to "keep"?

Reply:
@thisisdano Went ahead and updated this to "keep". Please let me know if you'd like any changes.